ui/src/i18n/sv.json

```diff
@@ -42,27 +42,10 @@
     "accessories.message_for_more_information": "för mer information.",
     "accessories.message_must_use_insecure_mode": "För att kontrollera dina tillbehör måste du köra Homebridge i <strong>osäkert</strong> läge.",
     "accessories.message_please_see": "Snälla se",
-    "accessories.message_rooms_not_appear_in_homekit": "The rooms you create here will not appear in HomeKit, they are only for the organisation of accessories in Homebridge Config UI X.",
+    "accessories.message_rooms_not_appear_in_homekit": "Rummen du skapar här visas inte i HomeKit, de är endast avsedda för organisering av tillbehör i Homebridge Config UI X.",
     "accessories.title_accessories": "Tillbehör",
     "accessories.title_accessory_control_disabled": "Homebridge Tillbehörs Kontroll Inaktiverad",
     "accessories.title_create_new_room": "Skapa nytt rum",
-    "backup.button_download_backup_archive": "Download Backup Archive",
-    "backup.button_restore_backup": "Restore Backup",
-    "backup.label_choose_backup_file_to_restore": "Choose backup file to restore...",
-    "backup.label_uploading": "Uploading...",
-    "backup.message_backup_archive_created": "Backup Archive Created",
-    "backup.message_backup_download_failed": "Backup Download Failed",
-    "backup.message_backup_help_one": "Download a backup archive of your entire Homebridge environment. This will backup the entire contents of your Homebridge storage directory which you can later restore on any platform capable of running Homebridge Config UI X.",
-    "backup.message_backup_help_two": "Plugins not hosted on npm, or plugins installed directly from GitHub will not be backed up or restored.",
-    "backup.message_backup_restored": "Backup Archive Restored",
-    "backup.message_backup_warning": "Backup archives contain sensative information and should not be shared with others.",
-    "backup.message_restore_failed": "Restore Failed",
-    "backup.message_restore_help_one": "Restore a backup archive you previously made using Homebridge Config UI X. Your Homebridge config, cached accessories, HomeKit pairings and Homebridge Config UI X user accounts will be restored. Any Homebridge plugins you had installed will also be downloaded from the npm registry.",
-    "backup.message_restore_help_two": "You will need to login to Homebridge Config UI X again after doing a restore using the credentials from the estored instance.",
-    "backup.message_restore_warning": "Restoring from a backup will irreversibly overwrite the current Homebridge configuration.",
-    "backup.title_backup": "Backup",
-    "backup.title_backup_restore": "Backup / Restore",
-    "backup.title_restore": "Restore",
```

Comment:
Thanks @DewGew for the PR.

Would you mind rebasing your branch to include these recently added messages?

Reply:
I noticed that to :)

Reply:
Fixed